py/requirements.txt

```diff
@@ -1,8 +1,8 @@
 numpy
 packaging
 pybind11==2.6.2
---extra-index-url https://download.pytorch.org/whl/nightly/cu124
-torch>=2.5.0.dev,<2.6.0
-torchvision>=0.20.0.dev,<0.21.0
+--extra-index-url https://download.pytorch.org/whl/test/cu124
+torch>=2.5.0,<2.6.0
+torchvision>=0.19.1,<0.20.0
```

Comment:
is 19.1 supported on 2.5?

Reply:
there is no 0.20.0 on 2.5, I have to downgrade to 0.19.1
[](https://download.pytorch.org/whl/test/torchvision/)